vtr_flow/benchmarks/titan_other_blif/README.rst

```diff
@@ -1,5 +1,7 @@
-The `Titan <http://www.eecg.utoronto.ca/~kmurray/titan/>` benchmarks
-are distributed seperately from VTR due to their large size.
+The `Titan <http://www.eecg.utoronto.ca/~kmurray/titan/>` benchmarks are distributed seperately from VTR due to their large size.
+
+The Titan repo is located under /home/kmurray/trees/titan on the U of T EECG network. Memebers of Vaughn Betz's research lab have read/write privileges. 
+
 
 To integrate them into VTR run:
```

Comment:
@ArashAhmadian : It would be good to explain what is on the web page vs. what is in the repo.  The web site is basically a mirror of the parts of the repo you need to run the Titan benchmarks through vtr. The repo is where any changes to the benchmarks should be made, along with scripts needed only to generate the Stratix IV architecture.

If there are other things I've forgotten in there please add them to the list (at a high level).

Reply:
Sounds good, will do.